docs/code/deployments-kubernetes/example.rego

```diff
@@ -1,15 +1,11 @@
 package example
 
-default deny = false
-
 # Reject objects without a customer label.
-deny {
-    not input.metadata.labels.customer
+deny["Missing customer label"] {
+    not input.tags.customer
 }
```

Comment:
It would be useful to also show a rule that returns a `true/false`

Reply:
The policy now matches the Docker example above. If we want to include true/false examples here, we could come back and add it later. Not sure it's required though.